pkg/controller/usersignup/approval_test.go

```diff
@@ -36,6 +36,8 @@ func TestGetClusterIfApproved(t *testing.T) {
 				MaxUsersNumber(2000, PerMemberCluster("member1", 1000)).
 				ResourceCapThreshold(80, PerMemberCluster("member1", 70)))
 		fakeClient := NewFakeClient(t, toolchainStatus, hostOperatorConfig)
+		InitializeCounters(t, fakeClient, toolchainStatus)
+
```

Comment:
in most of the cases, we need to initialize yet another client to use in the `initializeCounters` so we can call `counter.Synchronize(cl, toolchainStatus)`

Reply:
I know, but it's not needed for the evaluation of the test. The initialization is needed just to make the counter happy, so I wanted to make the initialization as easy as possible.

I forgot to mention, that I didn't want to reject this particular change - I just shared the reason why I did what I did: to not require the client when it's not needed and create the instance silently inside the function (it's just a implementation detail :smile:) . Sorry for not being clear.

Reply:
btw, the second part of my comment is still valid:

> In addition, you don't reset the counter, so it will be initialized only once - for the rest of the cases the initialization will be ignored.

so, if you want to make it consistent, you should also copy the reset to all test-cases

Reply:
addressed in https://github.com/codeready-toolchain/host-operator/pull/406/commits/cb6d5ddea48dbaa8f2cc23737a2bb788758785ab